modules/ROOT/pages/dataweave-formats.adoc

```diff
@@ -80,44 +80,36 @@ DataWeave supports the following formats as input and output:
 [[dw_readers_writers]]
 == DataWeave Readers
 
-DataWeave can read input data as a whole in-memory, in indexed fashion,
-and for some data formats, part-by-part by streaming the input. When attempting
-to read a large file, it is possible to run out of memory or to impact
-performance negatively. Streaming can improve performance but impacts access to file.
-
-* Indexed and In-Memory: +
-Allow for random access to data because both
-strategies parse the entire document. When using these strategies, a DataWeave
-script can access any part of the resulting value at any time.
-
-** Indexed: +
-Uses indexes over the disk.
-+
-DataWeave can only read the following formats in the indexed mode:
-+
-CSV, JSON, XML (starting in Mule 4.3.0)
-+
-Note that DataWeave _does not_ support reading Excel (XLSX) input in the indexed mode.
-
-For details, see xref:dataweave-indexed-readers.adoc[Indexed Readers in DataWeave].
-
-** In-Memory: +
-Parses the entire document in memory.
-+
-DataWeave can read all supported formats using the in-memory mode.
+DataWeave can read input data as a whole by loading it into memory or by indexing it in local storage and, for some data formats, read data sequentially in parts by streaming the input. The indexed reading strategy implements a mechanism to avoid out-of-memory issues when processing large files. The streaming strategy can improve performance but restricts file access to be sequentially, because you can access only the part of the file that is being read and stored in memory.
```

Comment:
```suggestion
DataWeave can read input data as a whole by loading it into memory or by indexing it in local storage and, for some data formats, DataWeave can read data sequentially in parts by streaming the input. The indexed reading strategy implements a mechanism to avoid out-of-memory issues when processing large files. The streaming strategy can improve performance but limits file access  sequential access (and not random access) because you can access only the part of the file that is being read and stored in memory.
```

Reply:
"to be sequentially," needed rewording for a grammar fix.